client/components/Game.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { progressBar } from '../helpers/gameHelpers.jsx';
+const Game = props => (
+  <div>
+    <div className="col-md-4 green">
+      <div> { props.name }</div>
+      <div className="progress">
+        <div className="progress-bar" role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100" style={{ width: `${progressBar(props.start, props.days, props.period, props.transactions)}%` }} /> 
```

Comment:
Small detail: It's usually a nice to break long single line html nodes with one line per attribute.